extension-support/test/common/deploy-phase-test.ts

```diff
@@ -211,4 +211,24 @@ describe('Deploy phase common module', () => {
             expect(storeParamStub.callCount).to.equal(1);
         });
     });
+
+    describe('Inject env vars from handel file', () => {
+        it('should inject env vars when properties are not specified', async () => {
+            const dependencyServiceContext = new ServiceContext<ServiceConfig>('fakeApp', 'fakeEnv', 'fakeService', new ServiceType('someExtension', 'dynamodb'), {type: 'dynamodb'}, accountConfig);
+            const dependencyDeployContext = new DeployContext(dependencyServiceContext);
+            dependencyDeployContext.addEnvironmentVariables({
+                TEST_DEPENDENCY_VAR: 'test_value2'
+            });
+            const dependenciesDeployContexts = [dependencyDeployContext];
+            const userProvidedEnvVars = {
+                TEST_USER_VAR: 'test_value'
+            };
+            const envVars = deployPhase.getEnvVarsForDeployedService(serviceContext, dependenciesDeployContexts, userProvidedEnvVars);
+            expect(envVars.HANDEL_REGION_NAME).to.equal('us-west-2');
```

Comment:
We should probably add some more expectations to verify that TEST_USER_VAR and TEST_DEPENDENCY_VAR were properly added to the 'envVars' variable.